docs/README.md

```diff
@@ -14,6 +14,7 @@ This directory is built automatically. Each task's documentation is generated fr
 * [Archive orders when fulfilled](./archive-orders-when-fulfilled)
 * [Archive orders when tagged](./archive-orders-when-tagged)
 * [Ask for reviews a week after order fulfillment](./ask-for-reviews-a-week-after-order-fulfillment)
+* [Auto publish products to sales channels by tag](./auto-publish-products-to-sales-channels-by-tag)
```

Comment:
Simplify to "Auto-publish products by tag" (applying that to the filename/slug as well).